addons/dango-rain/userscript.js

```diff
@@ -0,0 +1,100 @@
+let dangorain = false,
```

Comment:
Aren't these global now? What am I missing?

Reply:
No?? window.* makes things global.

Reply:
but it's not in a function so it inherits global scope right?

Reply:
@RedGuy12 Module context does not pollute the global scope implicitly, unlike non-module contexts.